content/SCALECLIReference/Auth/CLIApiKey.md

```diff
@@ -10,6 +10,193 @@ tags:
+API Key: 3-xTqwhyf3SrUgUlotMQEEGuUr6oRvqg89SBDfXob6xtWSgLbRiDBr6SVRWxswSXx3
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Delete Command
+
+The `delete` command deletes an API key.
+
+{{< expand "Deleting an API Key" "v" >}}
+
+#### Description
+The `delete` command requires the `id` property.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns nothing when successful.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>auth api_key delete id=<i>number</i></code>
+
+Where *number* is the list number of the API key you want to delete. For example, the first API key created on the system would be "1".
+{{< expand "Command Example" "v" >}}
+```
+auth api_key delete id=1
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Get_Instance Command
+
+The `get_instance` command returns a table of properties for the specified API key.
+
+{{< expand "Viewing API Key Instances" "v" >}}
+
+#### Description
+The `get_instance` command requires the `id` property.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns a table of properties for the specified API key when successful.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>auth api_key get_instance id=<i>number</i></code>
+
+Where *number* is the list number of the API key you want to delete. For example, the first API key created on the system would be "1".
+
+{{< expand "Command Example" "v" >}}
+```
+auth api_key get_instance id=2
++------------+---------------------------+
+|         id | 2                         |
+|       name | apikey2                   |
+| created_at | 2023-08-22T19:58:59+00:00 |
+|  allowlist | <list>                    |
++------------+---------------------------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Query Command
+
+The `query` command returns a table of properties for all API keys.
+
+{{< expand "Running an API Key Query" "v" >}}
+
+#### Description
+The `query` command has no requirements.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns a table of properties for all API keys when successful.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+`auth api_key query`
+
+{{< expand "Command Example" "v" >}}
+```
+auth api_key query
++----+---------+---------------------------+-----------+
+| id | name    | created_at                | allowlist |
++----+---------+---------------------------+-----------+
+| 2  | apikey2 | 2023-08-22T19:58:59+00:00 | <list>    |
+| 3  | apikey3 | 2023-08-22T20:14:59+00:00 | <list>    |
+| 4  | apikey4 | 2023-08-22T20:17:44+00:00 | <list>    |
++----+---------+---------------------------+-----------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Update Command
+
+The update command allows you to update existing API keys.
+
+{{< expand "Updating API Keys" "v" >}}
+
+#### Description
+The `update` command has two configurable properties and one argument. After specifying the `id` of the API key you want to update, you must include at least one of the configurable properties or arguments.
```

Comment:
What are the configurable properties for id in this instance? You can say something like this and include a table for the additional required properties for the update command, or if they are the same properties in the create command:
```suggestion
The `update` command has two configurable properties and one argument. 
Enter the `id` of the API key you want to update, and then you must include at least one of the configurable properties or arguments described in the <code>[create](#create-command)</code> command section.
```
If the update command uses different optional/required properties, what are they? Any specifics on how to enter the property argument? If the same as described in create then the pointer to that section covers it. :-)